src/BalanceTx.purs

```diff
@@ -530,6 +526,12 @@ balanceTx unattachedTx@(UnattachedUnbalancedTx { unbalancedTx: t }) = do
   feeBuffer :: BigInt
   feeBuffer = fromInt 500000
 
+-- Nami provides a 5 Ada collateral that we should add the tx before balancing.
```

Comment:
This comment should be updated to reflect the fact that all CIP-30 wallets do this now